### PR TITLE
Support Chinese language

### PR DESCRIPTION
**Pull request by eloiji**
Added translations to support chinese language

### CODE REVIEW

**Comment by devmikko** [comment]
Hi All, any update for this? :)

**Comment by bashmish** [comment]
@eloiji @devmikko we are going to discuss the guidelines for contributing new languages, I'll keep you updated. This is tricky because we normally have a professional translator to create translations.

In this case at least someone who speaks Chinese natively should check it, but we don't have anyone in the team and such an approach is not really scalable. So we gonna discuss how to make it more scalable and still have at least a reasonable check in place.

**Comment by bashmish** [comment]
@eloiji @devmikko regarding the workflow: after communication with @thatianamg we agreed that these translations are fine for now. We can accept your changes and in the meanwhile we are requesting translations from the external supplier.

Regarding the code: we have more components that need to be translated. In addition to the ones you translated we have `calendar`, `input-datepicker`, `input-iban`. If you are planning to use any of them, you need to provide translations for them as well, or otherwise there will be a runtime exception.

We are currently thinking about implementing a fallback locale in case translations are missing, so that instead of runtime exception components just display English (that we have everywhere) and developers can continue working on the code and contribute missing translations later. But this not how it works atm.

So I have 3 options for you to choose from:
1. You can add missing translations for `calendar`, `input-datepicker`, `input-iban` in this PR.
2. We keep them untranslated because you don't plan to use them right now, and later once we have English fallback feature, they will work, but just not translated.
3. I add translation files for `calendar`, `input-datepicker`, `input-iban`, but use English there, so that they work even without waiting for the English fallback feature. Practically I'll add this code:

```js
// in "zh.js" and "zh-CN.js"
import en from './en.js';

export default {
  ...en,
};
```

I think 1 would be awesome, 2 is acceptable since we are planning to add this feature soon, 3 is not a nice idea since it will conflict with the English fallback feature in the future, but we can accept this for now if you want these component to work, but don't have time to wait or add missing translation for Chinese.

Which one would you prefer?

**Comment by eloiji** [comment]
Hi @bashmish we prefer option 1.
I'll update this PR once we have the translations from our Chinese colleagues, for the other components. Thanks!

**Comment by bashmish** [comment]
@eloiji please also rebase with the latest master to not miss out some of the new translations which might be only there. And let me know when you are finished, looking forward to it 👍

**Comment by eloiji** [comment]
Hi @bashmish, Kindly review our changes. thanks!

File: packages/input-iban/translations/zh.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      'DE {German}\n' +
+      'ES {Spanish}\n' +
+      'FR {French}\n' +
+      'HU {Hungarian}\n' +
+      'IT {Italian}\n' +
+      'NL {Dutch}\n' +
+      'PL {Polish}\n' +
+      'RO {Romanian}\n' +
+      '另一个 {{validatorParams}}\n' +
+      '} {fieldName}。',
```
[the remark this answers is not in the document]
Comment (eloiji) [reply packages/input-iban/translations/zh.js]: I've updated the translations as reviewed by our PO.

**Comment by bashmish** [comment]
Thanks a lot for your help! 👍

**Pull request merged**